### PR TITLE
Post Carousel: Center Items setting

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve https://github.com/siteorigin/so-widgets-bundle/issues/1172

### CODE REVIEW

File: widgets/post-carousel/tpl/base.php [hunk 1]
```diff
@@ -34,6 +34,7 @@
 		     data-page="1"
 		     data-fetching="false"
 		     data-dir="<?php echo is_rtl() ? 'rtl' : 'ltr'; ?>"
+		     data-center-items="<?php echo esc_attr( $center_items ) ?>"
```
Comment (Misplon) [inline widgets/post-carousel/tpl/base.php:37]: Should this be `esc_attr( $center_items );` with a closing `;`?

**Comment by Misplon** [comment]
The `center-carousel` isn't outputting in my test. I have Posts per page set to 3.

carousel

Edit_Page_‹_SiteOrigin_—_WordPress

**Comment by Misplon** [comment]
The setting above this is "Loop posts" (we aren't using the word "items"). What do you think about changing the label and description to:

**Center posts**
Applies if the combined width of the visible posts is less than the carousel container width.

File: widgets/post-carousel/tpl/base.php (hunk 1 above)
Comment (AlexGStapleton) [reply widgets/post-carousel/tpl/base.php:37]: `;` isn't required in this situation and I left it out to match the previous line. Regardless, I'll change this (and the previous line).

**Comment by AlexGStapleton** [comment]
How does that look? Previous version would only center the posts if _all_ posts were showing (as in, you couldn't load more).

**Review by gregpriday — APPROVED** [review]

**Comment by Misplon** [comment]
@AlexGStapleton Is it expected that Posts per page isn't respected after the first load of additional posts?

**Review by Misplon — APPROVED** [review]

**Comment by AlexGStapleton** [comment]
Yes.
For the Post Carousel widget, the Posts Per Page setting label is is misleading and was the reason for [this PR](https://github.com/siteorigin/so-widgets-bundle/pull/1176). It's likely closer to "Posts Per Load" as it's how many posts that are loaded at a time. Are you okay with that label? If so, I'll submit a PR that renames Posts Per Page to that.

**Pull request merged**